src/Client/GroupsClient.php

```diff
@@ -63,7 +63,7 @@ public function buildApiUrl()
         return Curl::API_BASE_URL.'/'.Curl::API_VERSION.'/groups/'.$this->group->getKey().'/'.$this->action.'.json?token='.$this->group->getApplication()->getToken();
     }
 
-    public function buildCurlPostFields(Recipient $recipient = null): array
+    public function buildCurlPostFields(?Recipient $recipient = null): array
     {
         $curlPostFields = [
             'token' => $this->group->getApplication()->getToken(),
```

Comment:
just spotted, that in line 78, $recipient can be null, right?

Reply:
yes, can be `null`